llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -1145,6 +1146,95 @@ template <typename FrameIdTy> class CallStackRadixTreeBuilder {
   }
 };
 } // namespace memprof
+
+namespace yaml {
+template <> struct MappingTraits<memprof::Frame> {
+  static void mapping(IO &Io, memprof::Frame &F) {
+    Io.mapRequired("Function", F.Function);
+    Io.mapRequired("LineOffset", F.LineOffset);
+    Io.mapRequired("Column", F.Column);
+    Io.mapRequired("Inline", F.IsInlineFrame);
```

Comment:
Just noticed that the name of the field here is not consistent with the struct member name. Would it be better to make it consistent? (as a separate change if you think so)

Reply:
I borrowed the name from `Frame::printYAML`, which uses `"Inline"` for `IsInlineFrame`, but yes, I'm going to update it in a separate patch.  I think it's best to keep a move and a renaming separate.